history.md

```diff
@@ -3,33 +3,29 @@ layout: default
 css_id: history_timeline
```

Comment:
We should probably edit the css_id.  `history.md` once included the timeline.

Reply:
These comments so far look great. I was going to go forward and do a little more work ---there more to be added to both news and press. Should I modify what's here first? And do I keep everything on this pull request as I go?

Sorry if these are dumb questions. I generally do a pull request at the end of each editing session, so I'm never sure about re-using the same PR

Reply:
Yes, please keep everything on this pull request as you go.  In other words, continue pushing edits to this pull request.  This approach is easier for you and for reviewers.

Once you make edits that address one of my comments, my comment is "hidden."  I will also see your changes and can continue providing feedback.

Reply:
In the future, you can try focusing each pull request to a particular file to minimize its size.